lib/livebook_web/live/session_live.ex

```diff
@@ -49,6 +50,10 @@ defmodule LivebookWeb.SessionLive do
 
         session = Session.get_by_pid(session_pid)
 
+        if session.file do
+          SessionManager.save_recently_opened_sessions(session.file.path)
+        end
```

Comment:
We don't want to do this in the LV because it would trigger on every entry of every user. We should do this in `Session` init, and then when changing the file ([here](https://github.com/livebook-dev/livebook/blob/b8c19475a3084fbb6c943d600af09a1ef2d41a7f/lib/livebook/session.ex#L1676)) we want to remove the previous one and put the new one :)

Reply:
Probably to make it work across file systems, yes.